ssg/oval.py

```diff
@@ -119,7 +119,7 @@ def replace_external_vars(tree):
             sys.exit(2)
         # replace tag name: external -> local
         node.tag = "{%s}local_variable" % ovalns
-        literal = ET.Element("literal_component")
+        literal = ET.Element("oval:literal_component")
```

Comment:
Are you sure this works? Is the oval ns registered in this context? If this works I think it's undefined behavior and we are just lucky. We should do this with the proper API.

Reply:
@mpreisler then this issue is much larger and was created with https://github.com/ComplianceAsCode/content/pull/3185 which you acked. ;)

Reply:
nvm, there is a `ET.register_namespace("oval", ovalns)` call there.

Reply:
> @mpreisler then this issue is much larger and was created with #3185 which you acked. ;)

as you surely know, I never make mistakes :-DDD

Reply:
2014 mpreisler would.... but not 2018 mpreisler :D